Python/compile.c

```diff
@@ -2448,10 +2447,22 @@ compiler_jump_if(struct compiler *c, expr_ty e, basicblock *next, int cond)
         /* fallback to general implementation */
         break;
     }
-    default:
+    default: {
+        if (e->kind == Constant_kind) {
```

Comment:
Shouldn't this be it's on case in the previous switch?

(case Constant_kind: )

Reply:
Good point!